src/types/to_sql.rs

```diff
@@ -186,12 +218,27 @@ macro_rules! to_sql_self_fallible(
                 )))
             }
         }
+    );
+    (non_zero $t:ty) => (
+        impl ToSql for $t {
```

Comment:
I wanted to implement this by delegating to the zero-able type, but it doesn't seem possible because the `NonZero::get` returns by-value and `ToSql::to_sql` is allowed to borrow from the temporary.